package.json

```
@@ -15,7 +15,7 @@
   ],
   "repository": "expressjs/response-time",
   "dependencies": {
-    "depd": "~1.1.2",
+    "depd": "^2.0.0",
```

```suggestion
    "depd": "~2.0.0",
```

This will fix the CI I think